cloudify_cli/config/parser_config.py

```diff
@@ -19,52 +19,95 @@
 
 from cloudify_cli import commands as cfy
 from cloudify_cli.config import completion_utils
-from cloudify_cli.config import argument_utils
+from cloudify_cli.config.argument_utils import remove_type
+from cloudify_cli.config.argument_utils import remove_completer
+from cloudify_cli.config.argument_utils import make_required
+from cloudify_cli.config.argument_utils import make_optional
 from cloudify_cli.constants import DEFAULT_REST_PORT
+from cloudify_cli.constants import DEFAULT_BLUEPRINT_FILE_NAME
```

Comment:
I preferred it over writing `constants.` at the beginning of every constant in the file.